Documentation/recovery_v3.md

```diff
@@ -0,0 +1,60 @@
+## Disaster recovery
+
+etcd is designed to withstand machine failures. An etcd cluster automatically recovers from temporary failures (e.g., machine reboots) and tolerates up to _(N-1)/2_ permanent failures for a cluster of N members. When a member permanently fails, whether due to hardware failure or disk corruption, it loses access to the cluster. If the cluster permanently loses more than _(N-1)/2_ members then it disastrously fails, irrevocably losing quorum. Once quorum is lost, the cluster cannot reach consensus and therefore cannot continue accepting updates.
```

Comment:
prefer `*` to `_` for italics (https://github.com/coreos/docs/blob/master/STYLE.md#preferred-markdown-symbols)